x-pack/plugins/security_solution/cypress/tasks/alerts_detection_rules.ts

```diff
@@ -152,3 +160,15 @@ export const resetAllRulesIdleModalTimeout = () => {
   cy.window().trigger('mousemove', { force: true });
   cy.tick(700000);
 };
+
+export const changeRowsPerPageTo = (rowsCount: number) => {
+  cy.get(PAGINATION_POPOVER_BTN).click({ force: true });
+  cy.get(rowsPerPageSelector(rowsCount)).click();
+  waitForRulesTableToBeRefreshed();
+};
+
+export const goToPage = (pageNumber: number) => {
+  cy.get(RULES_TABLE_REFRESH_INDICATOR).should('not.exist');
+  cy.get(pageSelector(pageNumber)).last().click({ force: true });
+  waitForRulesTableToBeRefreshed();
+};
```

Comment:
What do you think about using this click-and-wait approach by default? Think it's safer and the tests would probably be less flaky.

Some of the tests here were not working because of the lack of proper waiting for certain indicators on the page to show up and then disappear. I added this waiting where needed: either like here, inside the tasks, or outside of tasks (which seems to be a more common pattern in our codebase).